src/components/delivery/Compass.js

```diff
@@ -41,8 +41,8 @@ const Compass = () => {
     }
   }
 
-  const longitude = posX;
-  const latitude = posY;
+  const longitude = posX.toFixed(6);
+  const latitude = posY.toFixed(6);
```

Comment:
Remember `posX` and `posY` could be `null`, so we may not be able to call `toFixed()` on them. Try `posX?.toFixed(6)` and `posY?.toFixed(6)` instead. It's called [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).